GVRf/Framework/framework/src/main/java/org/gearvrf/GVRBaseSensor.java

```diff
@@ -106,7 +106,9 @@ void processList(GVRCursorController controller) {
         if (events.isEmpty() == false) {
             final IEventReceiver ownerCopy = owner;
             for (SensorEvent event : events) {
-                eventManager.sendEvent(ownerCopy, ISensorEvents.class, "onSensorEvent", event);
+                if(event.getObject().isEnabled()) {
```

Comment:
Will be more efficient if we move this check to line 93 in SensorManager.java. This would result picking only being done on enabled objects.